classes/order/Order.php

```diff
@@ -34,6 +34,9 @@ class OrderCore extends ObjectModel
     /** @var int Delivery address id */
     public $id_address_delivery;
 
+    /** @var bool True if customer wants an invoice */
+    public $invoice = true;
```

Comment:
```suggestion
    public bool $invoice = true;
```
Since this is for the develop branch, you should be able to use all of php 8 advantages